src/pipx/main.py

```diff
@@ -195,6 +195,14 @@ def run_pipx_command(args: argparse.Namespace, subparsers: Dict[str, argparse.Ar
 
         venv_dir = venv_container.get_venv_dir(package)
         logger.info(f"Virtual Environment location is {venv_dir}")
+
+    if "packages" in args:
+        if any(urllib.parse.urlparse(package).scheme for package in args.packages):
```

Comment:
BTW, this doesn't validate whether a package is a URL correctly. It should at least also check whether it contains a network location. The current form would detect a Windows path as a URL. See the [docs](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlparse).

Reply:
Hi @chrysle, any suggestions to check it more precisely?

Reply:
I've encountered this just recently and fixed it in https://github.com/pypa/pipx/pull/1329/commits/e94c5d2a92338012a12067ba049182d72aaa1f35.

Reply:
Hi @chrysle, seems the commit is not merged yet, should I send a pull request for this or for the `packages` part only?

Reply:
Just a reminder, seems the #1329 is merged. Maybe I should also make the same change to packages?

Reply:
Let's do it.